src/main/asciidoc/reference/mapping.adoc

```diff
@@ -59,6 +59,186 @@ The following outlines what type conversion, if any, will be done on the propert
 
 When querying and updating `MongoTemplate` will use the converter to handle conversions of the `Query` and `Update` objects that correspond to the above rules for saving documents so field names and types used in your queries will be able to match what is in your domain classes.
 
+[[mapping-conversion]]
+== Data mapping and type conversion
+
+This section explain how types are mapped to a MongoDB representation and vice versa. Spring Data MongoDB supports all types that can be represented as BSON, MongoDB's internal document format.
+In addition to these types, Spring Data MongoDB provides a set of built-in converters to map additional types. You can provide your own converters to adjust type conversion, see <<mapping-explicit-converters>> for further details.
+
+[cols="3,1,6", options="header"]
+.Type
+|===
+| Type
+| Type conversion
+| MongoDB representation
```

Comment:
Should we rather call this _Sample_ as the actual representation of a `GeoJsonPoint` would just be `{ "type" : "Point" , "coordinates" : [3.0 , 4.0] }`.